blockstore/apps/api/v1/views/bundles.py

```diff
@@ -12,6 +15,21 @@
 from ..serializers.bundles import BundleSerializer, BundleVersionSerializer, BundleVersionWithFileDataSerializer
 
 
+class BundleFilter(FilterSet):
+    """
+    Filter for BundleViewSet, which allows filtering by multiple at once UUIDs
+    """
+    uuid = AllValuesMultipleFilter(widget=CSVWidget)
```

Comment:
If one of the options is an id which no longer exists in the database, for example when requesting:

`http://localhost:18250/api/v1/bundles?uuid=da5f15da-d837-42ca-89e9-a29de2bc7a99,bffafb3f-1ec3-4d47-9a89-a12c4a87db3b`

The error message seems to not contain the invalid value correctly. Can this be fixed?

```
{"uuid":["Select a valid choice. %(value)s is not one of the available choices."]}
```

Reply:
Thanks for noticing. It was a bug in `django-filters`, fixed by bumping the version.

Reply:
I think the version bump needs to be pushed? I don't see it in the diff.

Reply:
Oops, my bad. Fixed!

Reply:
Great, thanks!